### DISCUSSION TITLE
Animal Crossing: New Horizons

### DISCUSSION THREAD

**Discussion by D-06** [skyline-emu/skyline#584] [Game Requests]
Maybe will work someday

**Comment by DBXJes** [comment]
Here's an actual picture of the last version. (Not working). 0.0.3 App Android.

![Screenshot_20220929-005744_Skyline.jpg](https://user-images.githubusercontent.com/64260560/192902964-91000a3d-3c6e-4838-bf9e-a5cd61a71d37.jpg)
